libs/TaskManager.js

```diff
@@ -149,6 +153,19 @@ module.exports = class TaskManager{
 			if (task){
 				this.addToRunningQueue(task);
 				task.start(() => {
+					if(task.webhook && task.webhook.length > 3){
```

Comment:
I see! I was just curious. Would the program behave correctly (not crash, or print an error message) if I created a task with a webhook value of "vjkljlkuicx" (not a URL)?